thavalon-webapp/src/styles/gameStyles/gameGlobals.scss

```diff
@@ -47,3 +53,11 @@ $good-role-color: #003A06;
     color: $game-section-header-color;
     text-align: center;
 }
+
+// Title of any subheader in a game
+.game-section-subheader {
+    font-size: 20px;
+    line-height: 26px;
```

Comment:
i messed this up in a bunch of places but ideally we should be using em/rem, but maybe we should do one pass over this once everything else is done.

Reply:
Yeah I just copied and pasted. We can go over it and swap out.